streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilderTest.java

```diff
@@ -553,56 +547,6 @@ public void shouldAddInternalTopicConfigForRepartitionTopics() {
         assertTrue(topicConfig instanceof RepartitionTopicConfig);
     }
 
-    @Test
-    public void shouldThrowOnUnassignedStateStoreAccess() {
```

Comment:
This test is removed since we already have the exact same test in `TopologyTest`.